.travis.yml

```diff
@@ -31,6 +31,7 @@ env:
   - CXX=g++-4.8 NODE_ENV=test DB=mysql
 
 before_install:
+  - npm i -g npm@6
```

Comment:
Not sure if it matters (or if you want the latest npm@6 version), but Node v8.12 includes npm 6.4.1, per https://nodejs.org/en/download/

> Latest LTS Version: 8.12.0 (includes npm 6.4.1)